app/src/main/java/com/android/shelfLife/MainActivity.kt

```diff
@@ -47,6 +50,8 @@ fun ShelfLifeApp() {
   val listFoodItemViewModel = ListFoodItemsViewModel(foodItemRepository)
   val householdViewModel =
       HouseholdViewModel(HouseholdRepositoryFirestore(firebaseFirestore), listFoodItemViewModel)
+  val foodFactsRepository = OpenFoodFactsRepository(OkHttpClient())
+  val foodFactsViewModel = FoodFactsViewModel(foodFactsRepository)
```

Comment:
ViewModels should be instantiated using the viewModel() composable to ensure proper lifecycle management. Instantiating it as a normal class can lead to issues like data loss during configuration changes or recompositions.

The official documentation explain quite well how to work with viewModel: https://developer.android.com/topic/libraries/architecture/viewmodel#implement-viewmodel